src/team_lib/hier/xccl_hier_context.c

```diff
@@ -147,7 +158,7 @@ xccl_status_t xccl_hier_create_context(xccl_team_lib_t *lib,
     ctx->bcast_sm_get_thresh                    = hier_cfg->bcast_sm_get_thresh;
 
     ucs_for_each_bit(tl, XCCL_TL_ALL) {
-        if (XCCL_OK != xccl_hier_init_tl(ctx, tl, params->oob, config->env_prefix)) {
+        if (XCCL_OK != xccl_hier_init_tl(ctx, tl, params->oob, hier_cfg)) {
             return XCCL_ERR_NO_MESSAGE;
```

Comment:
return actual error instead of XCCL_ERR_NO_MESSAGE

Reply:
done